test/jdk/javax/swing/JComponent/7154030/bug7154030.java

```diff
@@ -98,10 +98,11 @@ public void run() {
             robot.waitForIdle(1000);
 
             Rectangle bounds = frame.getBounds();
-            locx = bounds.x;
-            locy = bounds.y;
-            frw = bounds.width;
-            frh = bounds.height;
+            Insets insets = frame.getInsets();
+            locx = bounds.x + insets.left;
+            locy = bounds.y + insets.top;
+            frw = bounds.width - insets.left - insets.right;
+            frh = bounds.height - insets.top - insets.bottom;
```

Comment:
If the intent is to do away with window decorations, can't we just call frame.setUndecorated instead of this snippet?
Also, it will be better to post a mach5 job in JBS running several iterations as it fails in mach5 primarily.

Reply:
Or even check the location of the button only?

Reply:
This is the simple fix and i don't want to make the analysis are too small because this will make artifacts harder to analyze. Test runs are linked in the bug comment.